client/catapult/docs/BUILD-conan.md

```diff
@@ -92,13 +92,13 @@ CONAN_REVISIONS_ENABLED=1 conan install .. --build missing
 * Generate project files for Visual Studio 2022:
 
   ```sh
-  cmake -G "Visual Studio 17 2022" -A x64 -DUSE_CONAN=ON -DPYTHON_EXECUTABLE:FILEPATH=X:/python3x/python.exe ..
+  cmake --preset conan-default -G "Visual Studio 17 2022" -A x64 -DUSE_CONAN=ON -DPYTHON_EXECUTABLE:FILEPATH=X:/python3x/python.exe ..
   ```
 
 * Generate project files for Visual Studio 2019:
 
   ```sh
-  cmake -G "Visual Studio 16 2019"  -A x64 -DUSE_CONAN=ON -DPYTHON_EXECUTABLE:FILEPATH=X:/python3x/python.exe ..
```

Comment:
this seems like old VS version now 😅

Reply:
Yeah 😅, but I left it since we always support the last two versions of the compiler.
We don't need both in the docs if you want to remove it.